homeassistant/components/media_player/kodi.py

```diff
@@ -523,9 +531,20 @@ def supported_features(self):
         if self._turn_off_action in TURN_OFF_ACTION:
             supported_features |= SUPPORT_TURN_OFF
 
+        if self._mac:
+            supported_features |= SUPPORT_TURN_ON
+
         return supported_features
 
     @cmd
+    def async_turn_on(self):
```

Comment:
Sure can.
Wasn't sure about this anyways (never worked with asyncio and this is the first time me working with hass)
Want me to rebase and force-push or do you guys prefer multiple commits?